markdown_parser.leg

```diff
@@ -223,6 +225,23 @@ BlockQuoteRaw =  a:StartList
                      $$->key = RAW;
                  }
 
+CommentBlock = a:CommentBlockRaw
```

Comment:
Comment Block hat Prio über inline comments und comments sehen außerhalb von Ulysses alle gleich aus.

Exportiert man also einen inline comment, der isoliert in einem Absatz steht, wird dieser als block comment importiert. Denke das ist okay.